docs/wmco-prerequisites.md

```diff
@@ -2,8 +2,16 @@
 Below are the Cluster and OS prerequisites for WMCO. Please also see the vSphere-specific section that 
 may be relevant.
 
-## Supported Cloud Providers based on OKD/OCP Version and WMCO version
-| Cloud Provider            | Supported OKD/OCP Version | Supported WMCO version |
+## Supported Installation method
+* Installer-Provisioned Infrastructure (IPI) installation method is the recommended and supported installation method. 
+  The latter is consistent across all supported platforms. 
```

Comment:
`The latter` can be used after two options given, to refer to the final option. since only IPI is given at this point it wouldn't be correct to use that. Instead I recommend rephrasing to:
```
 * Installer-Provisioned Infrastructure (IPI) installation method is fully supported and is the recommended installation method.
```

Reply:
True, makes it more straightforward.  Updated